File: developer-quarkus/src/main/java/de/dkutzer/buggy/developer.java
package de.dkutzer.buggy;

import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RestController;
import org.springframework.web.bind.annotation.PathVariable;


@RestController
@RequestMapping("/hello")
public class developer {

    @GetMapping
    public String hello() {
        return "hello";
    }
}

File: developer/src/main/kotlin/de/dkutzer/buggy/developer/developer/entity/Developer.kt
package de.dkutzer.buggy.developer.developer.entity

import org.springframework.data.annotation.Id
import org.springframework.data.annotation.TypeAlias
import org.springframework.data.mongodb.core.mapping.Document

@Document(collection = "developers")
@TypeAlias(value = "developer")
data class Developer (
        @Id
        val id: String,
        val firstName: String,
        val lastName: String
)
